laos_climate_change/data_loaders.py

```diff
@@ -0,0 +1,29 @@
+import os
+from os.path import exists
+from urllib.request import urlretrieve
+
+world_url = "https://datacatalogfiles.worldbank.org/ddh-published/0038272/DR0046659/wb_countries_admin0_10m.zip?versionId=2024-05-14T14:58:01.5696428Z"
+world_filename = "wb_countries_admin0_10m.zip"
+laos_url = "https://data.humdata.org/dataset/9eb6aff1-9e3f-43d3-99a6-f415fe4b4dff/resource/1d6edf99-5303-4b31-8909-dd70cda78443/download/lao_adm_ngd_20191112_shp.zip"
+laos_filename = "lao_adm_ngd_20191112_shp.zip"
+
+
+def download_shapefile(
+    which, output_path=".../laos-climate-change/laos_climate_change"
```

Comment:
I suggest that inside `data` we should have a `shapefiles` sub-directory. So I'd save these to `data/shapefiles/blah_blah.zip`

Reply:
> Output path needs to be in data, not in `laos_climate_change`.
> 
> All data goes in `data/`. `laos_climate_change/` is our python package where code lives.

Ok. Two questions:
1. Just to confirm: the folder with the scripts to process emdat and download the shapefiles should be into /data, not into the repository?

2  But, shouldn't we have a a more informative name for laos_climate_change? like scripts_laos_climate_change? Specially because the repo folder is called laos-climate-change, so it is very easy to confuse them